api-get-object.go

```diff
@@ -97,8 +161,8 @@ func (c Client) GetObject(bucketName, objectName string) (*Object, error) {
 			}
 		}
 	}()
-	// Return the readerAt backed by routine.
-	return newObject(reqCh, resCh, doneCh, objectInfo), nil
+	// Create a newObject through the information sent back by firstReq.
```

Comment:
Can you word this differently.. ? - newObject is already created regardless of information sent back by firstReq.